src/sentry/static/sentry/app/actionCreators/sentryAppTokens.jsx

```diff
@@ -0,0 +1,51 @@
+import {
+  addErrorMessage,
+  addLoadingMessage,
+  addSuccessMessage,
+  clearIndicators,
+} from 'app/actionCreators/indicator';
+import {t} from 'app/locale';
+
+/**
+ * Install a sentry application
+ *
+ * @param {Object} client ApiClient
+ * @param {Object} app SentryApp
+ */
+export async function addSentryAppToken(client, app) {
+  addLoadingMessage();
+  try {
+    const resp = await client.requestPromise(`/sentry-apps/${app.slug}/api-tokens/`, {
+      method: 'POST',
+    });
+    clearIndicators();
+    return resp;
+  } catch (err) {
+    addErrorMessage(t('Unable to create token'));
+    throw err;
+  }
+}
+
+/**
+ * Uninstall a sentry application
+ *
+ * @param {Object} client ApiClient
+ * @param {Object} app SentryApp
+ * @param {String} token Token string
+ */
+export async function removeSentryAppToken(client, app, token) {
+  addLoadingMessage();
+  try {
+    const resp = await client.requestPromise(
+      `/sentry-apps/${app.slug}/api-tokens/${token}/`,
+      {
+        method: 'DELETE',
+      }
+    );
```

Comment:
Formatting nit: we should make this formatting match the `addSentryAppToken` method, so the file's easier to skim though. ie:

```javascript
const resp = await client.requestPromise(`/sentry-apps/${app.slug}/api-tokens/${token}/`, {
    method: 'DELETE',
})
```